flutter_app/lib/screens/suppliers/components/list_components/supplier_empty_state.dart

```diff
@@ -1,45 +1,90 @@
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
 import 'package:shadcn_ui/shadcn_ui.dart';
 
 import '../../../../services/internationalization/internationalization.dart';
+import '../../../../widgets/pop_up/add_category/add_category_view.dart';
```

Comment:
isn't there any rule to enforce or prefer use of absolute path instead of relative path? I think it will makes the import more clear and understandable.
[just a point of view]

Reply:
> isn't there any rule to enforce or prefer use of absolute path instead of relative path? I think it will makes the import more clear and understandable. [just a point of view]

we don't have this rule, I will look for adding that.